logger/module.go

```diff
@@ -9,18 +10,18 @@ import (
 )
 
 type loggerModule struct {
+	zerolog.Logger
 	bus    modules.Bus
-	logger modules.Logger
 	config modules.LoggerConfig
 }
 
-// All loggers branch out of mainLogger, that way configuration changes to mainLogger propagate to others.
-var mainLogger = zerolog.New(os.Stderr).With().Timestamp().Logger()
-
 // The idea is to create a logger for each module, so that we can easily filter logs by module.
 // But we also need a global logger, because sometimes we need to log outside of modules, e.g. when the process just
 // started, and modules are not initiated yet.
-var Global = new(loggerModule).CreateLoggerForModule("global")
+var Global = loggerModule{
+	// All loggers branch out of mainLogger, that way configuration changes to mainLogger propagate to others.
+	Logger: zerolog.New(os.Stdout).With().Timestamp().Logger(),
```

Comment:
Wdyt of initialising `Global` inside an `init` function?

Ref: https://tutorialedge.net/golang/the-go-init-function/

Reply:
Makes sense, will add an init function !

Reply:
Bump

Reply:
Init function created